File: Basic/Java/src/samples/QiStreamMode.java
package samples;

public enum QiStreamMode {

    Continuous(0),
    StepwiseContinuousLeading(1),
    StepwiseContinuousTrailing(2),
    Discrete(3);
    private final int QiStreamMode;

    private QiStreamMode(int id) {
        this.QiStreamMode = id;
    }

    public int getValue() {
        return QiStreamMode;
    }
}

File: Basic/DotNet/QiRestApiCore/QiStreamMode.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace QiRestApiCore
{
    public enum QiStreamMode
    {
        Default = Continuous,
        Continuous = 0,
        StepwiseContinuousLeading = 1,
        StepwiseContinuousTrailing = 2,
        Discrete = 3
    }
}
